NEWS.md

```diff
@@ -4,6 +4,8 @@ Julia v1.12 Release Notes
 New language features
 ---------------------
 
+* The `@ccall` macro can now take a `gc_safe` argument, that if set to true allows the runtime to run garbage collection concurrently to the `ccall`
```

Comment:
Should probably have been in HISTORY since you want it backported

Reply:
Addressed by #57679 and #57682